File: frontend/src/app/dtos/hall-rows-detail-dto.ts
import { LocationDetailDto } from './location-detail-dto';
import { RowListDto } from './row-list-dto';

export interface HallRowsDetailDto {
  id: number;
  name: string;
  standingCount: number;
  location: LocationDetailDto;
  rows: RowListDto[];
}

File: backend/src/main/java/at/ac/tuwien/sepr/groupphase/backend/endpoint/dto/HallRowsDetailDto.java
package at.ac.tuwien.sepr.groupphase.backend.endpoint.dto;

import lombok.AllArgsConstructor;
import lombok.EqualsAndHashCode;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;
import lombok.ToString;

@Getter
@Setter
@EqualsAndHashCode
@AllArgsConstructor
@NoArgsConstructor
@ToString
public class HallRowsDetailDto {

    private Long id;

    private String name;

    private Long standingCount;

    private LocationDetailDto location;

    private RowListDto[] rows;

}
